vehicle-isabelle/.gitignore

```diff
@@ -0,0 +1 @@
+*~
```

Comment:
Rather than adding lots of extra `.gitignore` files scattered throughout, can we add these to the top-level .gitignore file?